docs/tasks/configure-pod-container/projected-volume.md

```diff
@@ -8,35 +8,41 @@ redirect_from:
 - "/docs/user-guide/projected-volume/index.html"
 ---
 
-The _projected volume_ is a volume that projects several existing volume sources
-into the same directory. Currently, one can project configmaps, downward API,
-and secrets. The resulting pod spec is also shorter when projecting to a single
-volume as opposed to multiple different locations. See [all-in-one volume design document](https://github.com/kubernetes/community/blob/{{page.githubbranch}}/contributors/design-proposals/all-in-one-volume.md)
-for more information.
+{% capture overview %}
+This page shows how to use a [`projected`](/docs/concepts/storage/volumes/#projected) volume to mount several existing volume sources into the same directory. Currently, `secret`, `configMap`, and `downwardAPI` volumes can be projected.
 
-* TOC
-{:toc}
+{% endcapture %}
 
-## Overview of a projected volume
+{% capture prerequisites %}
 
-The projected volume encapsulates multiple volumes to be projected, with each
-volume source respecting nearly the same parameters as supported by each
-individual type. Consider the following example:
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Configure a pod to project multiple sources into a single directory
+
+In this exercise, you create a Pod that runs Redis in one Container. The Pod uses two types of Volumes: [`emptyDir`](/docs/concepts/storage/volumes/#emptydir) is used for storing Redis data, and [`projected`](/docs/concepts/storage/volumes/#projected) mounts multiple secrets into the same shared directory. Here is the configuration file for the Pod:
```

Comment:
I'm on the fence about adding anything about Redis, but perhaps it makes the scenario more realistic. In such a case, I think it's best to mention creating the secrets is a prerequisite for the pod to start.